core/rfb.js

```diff
@@ -2941,6 +2936,35 @@ export default class RFB extends EventTargetMixin {
             "raw", passwordChars, { name: "DES-ECB" }, false, ["encrypt"]);
         return legacyCrypto.encrypt({ name: "DES-ECB" }, key, challenge);
     }
+
+    static convertButtonMask(buttons) {
```

Comment:
Given that it's an internal function, as is `_handleMouse()`, can't we put them next to each other to make it easier to read?